### ISSUE TITLE
enabled applehda without hotpatch and clover

### ISSUE THREAD

**Issue by chinieer** [onejay09/osx#2]
i just figure out that is useless to inject audio in clover and i change layoutid to 27 can solve the problem with headphone output unusual

**Comment by onejay09** [comment]
yes the layout id is set to 27 in my config.plist, i looked at AppleALC source and tried all asus specific layout id's for alc255
